src/ParametersHandler/TomlImplementation/include/BipedalLocomotion/ParametersHandler/TomlImplementation.h

```diff
@@ -220,7 +220,15 @@ class TomlImplementation : public IParametersHandler
      * @return A pointer to IParametersHandler, if the group is not found the weak pointer cannot
      * be locked
      */
-    weak_ptr getGroup(const std::string& name) const final;
+    const_weak_ptr getGroup(const std::string& name) const final;
+
+    /**
+     * Get a Group from the handler.
+     * @param name name of the group
+     * @return A pointer to IParametersHandler, if the group is not found the weak pointer cannot
+     * be locked
+     */
```

Comment:
I would suggest adding some details on the fact that this group can be edited afterwards